source/includes/aggregation.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/builder/basic/kvp.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/pipeline.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main() {
+    mongocxx::instance instance{};
```

Comment:
```suggestion
    mongocxx::instance instance;
```

Minor: but latest suggestion is to omit `{}`. See https://github.com/mongodb/docs-cpp/pull/25#discussion_r1683098263

Reply:
I will create a follow-up PR that removes the `{}` from all instances in the repo